addons/cassandra/1.x/cassandra-2.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: cassandra
+  labels:
+    kubeaddons.mesosphere.io/name: cassandra
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "3.11.7-1"
+    appversion.kubeaddons.mesosphere.io/cassandra: "3.11.7"
+    catalog.kubeaddons.mesosphere.io/kudo-params: "https://raw.githubusercontent.com/mesosphere/kudo-cassandra-operator/v3.11.7-1.0.2/operator/params.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.16.0
+  requires:
+    - matchLabels:
+        # requires servicemonitors.monitoring.coreos.com CRD to be present
+        kubeaddons.mesosphere.io/name: prometheus
+  cloudProvider:
```

Comment:
If you're not intentionally excluding gcp, you can remove the cloudProvider section and it will be enabled for all (the default behavior). If you are, I'd prefer to see it listed with `enabled: false`.

Reply:
Thanks for pointing this out @joejulian ! I think GCP should be enabled as I don't see any background why thiis addon won't work on GCP.